examples/examples/sloggish/sloggish_subscriber.rs

```diff
@@ -249,6 +248,7 @@ impl Subscriber for SloggishSubscriber {
                 self.print_kvs(&mut stderr, data.kvs.iter().map(|(k, v)| (k, v)), "")
                     .unwrap();
             }
+            #[allow(clippy::write_with_newline)]
             write!(&mut stderr, "\n").unwrap();
```

Comment:
Why not just change this to 
```suggestion
            stderr.write_char('\n').unwrap()
```
and avoid having to add the `allow` attribute?